src/args.jl

```diff
@@ -1638,8 +1643,7 @@ function _update_series_attributes!(plotattributes::AKW, plt::Plot, sp::Subplot)
 
     # set label
     label = plotattributes[:label]
-    label = (label == "AUTO" ? "y$globalIndex" : label)
-    label = label in (:none, nothing, false) ? "" : label
+    label = label_to_string(label, globalIndex)
     plotattributes[:label] = label
```

Comment:
Instead of the three lines above, you could just do `plotattributes[:label] = label_to_string(plotattributes[:label], globalIndex)`. Both is fine for me though.

Reply:
I was doing the same thing, but then thought it will complicate the code :sweat_smile: